src/main/java/org/apache/flink/benchmark/SerializationFrameworkMiniBenchmarks.java

```diff
@@ -158,7 +158,7 @@ public LongStringSource(int numEvents, int numKeys) {
 		}
 
 		@Override
-		protected void init() {
+		protected void init() throws Exception {
```

Comment:
In general, I'm not a big fan of Pokemon exception handling (gonna catch them all). Could we make it more specific (e.g. only IOException). 
I see the point that it makes implementing new sources much easier, so this comment is more of a nice-to-have.